test/scripts/run-e2e-tests.sh

```diff
@@ -122,6 +122,15 @@ kubectl apply --filename https://github.com/knative/serving/releases/download/${
 echo "Waiting for knative started ..."
 waiting_pod_running "knative-serving"
 
+echo "Installing cert manager ..."
+kubectl create namespace cert-manager
```

Comment:
Let's skip this by default

Reply:
If we need it - shouldn't it be installed by default? Unless the default behaviour is that kfserving works without a cert?

Reply:
I think we should have cert manager as part of the standalone kfserving installation and if kubeflow already has cert manager we can simply skip that and assume it is there.